client/app/telecom/telephony/alias/portability/portabilities/translations/Messages_fr_FR.xml

```diff
@@ -33,4 +33,10 @@
    <translation id="telephony_alias_portabilities_step_duration_day" qtlid="392450">jour(s)</translation>
    <translation id="telephony_alias_portabilities_step_duration_hour" qtlid="8902">heure(s)</translation>
    <translation id="telephony_alias_portabilities_step_duration_openday" qtlid="392463">jour(s) ouvré(s)</translation>
+   <translation id="telephony_alias_portabilities_cancel">Annuler la portabilité ?</translation>
+   <translation id="telephony_alias_portabilities_cancel_confirm">Etes-vous sûr de vouloir annuler votre demande de portabilité ?</translation>
+   <translation id="telephony_alias_portabilities_cancel_doing">Annulation en cours...</translation>
```

Comment:
`...` => `…`

Reply:
One day you will tell me how we can type this with the keyboard :-) Tired to copy/paste :-p